benchmark.py

```
@@ -0,0 +1,833 @@
+        if exc_type is not None:
+            self.print_error_log()
+
+    async def wait_for_ready(self, timeout: float | None = None) -> None:
+        if timeout is None:
+            timeout = self.config.server_ready_timeout
+        deadline = time.time() + timeout
+        while True:
+            url = f'http://localhost:{self.port}/_health'
+            try:
+                async with aiohttp.ClientSession() as session:
+                    async with session.get(url) as res:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)